include/zephyr/bluetooth/audio/bap.h

```diff
@@ -50,6 +50,63 @@ extern "C" {
 /** An invalid Broadcast ID */
 #define BT_BAP_INVALID_BROADCAST_ID 0xFFFFFFFFU
 
+/**
+ * @name Suggested connection and advertising intervals suited for coexistence with audio
+ *
+ * Suggest use is to use this as both the minimum and maximum interval for best results.
+ * The intervals are suited for both unicast and broadcast.
+ * @{
+ */
+/**
+ * @brief Fast interval suited for coexistence with 10ms SDU intervals
+ *
+ * This works well with ISO intervals of 10, 20 and 30 milliseconds for broadcast and unicast
+ */
+#define BT_BAP_COEX_INT_MS_10_FAST_1   60U
+/**
+ * @brief Fast interval suited for coexistence with 10ms SDU intervals
+ *
+ * This works well with ISO intervals of 10, 20, 30 and 40 milliseconds.
+ */
+#define BT_BAP_COEX_INT_MS_10_FAST_2   120U
```

Comment:
Should we have some documentation on why these values are better suited for these intervals, and when to use FAST_1 vs. FAST_2? Not necessarily here, but maybe in the samples?

Reply:
I can further describe in the group description - Good point